syntax = "proto3";

message AigerCircuit { string circuit = 1; }

message AigerBinaryCircuit { string circuit = 1; }